doc/API/centreon-api-v2.yaml

```diff
@@ -469,7 +469,6 @@ paths:
         * type
         * status_code
         * status
-        * action_url
```

Comment:
filter on action_url seems be possible : https://github.com/centreon/centreon/pull/9188/files#diff-c2589173fdcecc9bb1272eb92e1e134059d00e73532cf9f4d7074cf0006757c7R77

Reply:
Ok should I keep it ? And also since I've changed the code related to action_url I might do more changes for this

Reply:
Ok I've restored in the documentation the search by action_url :+1: